packages/cashier/src/Components/Form/crypto-withdraw-form.tsx

```diff
@@ -157,13 +210,12 @@ const CryptoWithdrawForm = ({
                                     type='submit'
                                     primary
                                     large
-                                    onClick={() => requestWithdraw(verification_code)}
```

Comment:
replaced `onClick` on `onSubmit` method because Formik types must have required properties : `initialValues` and `onSubmit`. That’s why i did this small refactoring regarding submission the form.